linux/iscsi.go

```diff
@@ -193,9 +233,8 @@ func HandleIscsiDiscovery(volume *model.Volume) (err error) {
 		return err
 	}
 
-	if !loggedIn {
```

Comment:
looks like you can directly call updateChapFoLoggedInTargets() here in else case instead of passing to another function.

Reply:
sure